### PR TITLE
Perfdash: requesting builds data on demand

### PR DESCRIPTION
**Pull request by krzysied**
Instead of getting all available data for every possible <job x test>, only requested data (for one selected <job x test>) will be retrieved.
ref #137

### CODE REVIEW

File: perfdash/www/script.js [hunk 1]
```diff
@@ -41,14 +41,14 @@ PerfDashApp.prototype.onClickInternal_ = function(data, evt, chart) {
...
+    this.http.get("jobnames")
             .success(function(data) {
-                this.jobNames = Object.keys(data);
+                this.jobNames = data;
                 //init jobName only if needed
                 if (this.jobName == undefined || this.jobNames.indexOf(this.jobName) == -1) {
                     this.jobName = this.jobNames[0];
                 }
-                this.allData = data;
+                //this.allData = data;
```
Comment (shyamjvs) [inline perfdash/www/script.js]: please remove this commented line

**Comment by shyamjvs** [comment]
One nit and lgtm

File: perfdash/www/script.js (hunk 1 above)
Comment (krzysied) [reply perfdash/www/script.js]: done

**Pull request merged**